cardano-cli/src/Cardano/CLI/EraBased/Options/Transaction.hs

```diff
@@ -53,6 +53,10 @@ pTransactionCmds era envCli =
         $ subParser "sign"
         $ Opt.info (pTransactionSign envCli)
         $ Opt.progDesc "Sign a transaction"
+    , Just
+        $ subParser "echo"
+        $ Opt.info pTransactionEcho
+        $ Opt.progDesc "Echo a transaction"
```

Comment:
Can you provide more context as to when this command is useful?

Reply:
Yep, I don't understand why we need this command.

Reply:
Yes, reading and saving the transaction again and verifying that the new transaction is identical to the old.

Reply:
Related to this investigation.  https://input-output-rnd.slack.com/archives/CR599HMFX/p1712568383003499

Reply:
Sooo... what's the advantage over `transaction view` ?

Reply:
`transaction view` doesn't do a roundtrip.  It reads a transaction and writes it in some other format.

Reply:
This looks like a thing we should have a property test for.

Reply:
So should we have a roundtrip test instead of a command?

Reply:
I guess both? The command could be useful too I think.